dashboard/src/lib/store.ts

```diff
@@ -0,0 +1,4 @@
+import type { Writable } from 'svelte/store'
+import { writable } from 'svelte/store'
+
+export const progress: Writable<number> = writable(0)
```

Comment:
I would give progress a more meaningful name, something like `deviceCreationProgress`, not very scalable name but I think it improves readability